packages/lingui-react/src/WithI18n.js

```diff
@@ -16,6 +16,12 @@ export default ({ update = true }: WithI18nOptions = {}) => function<P, C: React
       i18nManager: React.PropTypes.object
     }
 
+	ctrls={wrappedInstance:null}
+   
+    getWrappedInstance(){
+		return this.ctrls.wrappedInstance
```

Comment:
Feel free to add `wrappedInstance` property directly to instance as `react-redux` does it.